### PR TITLE
Feature/qr code scanner

### PR DESCRIPTION
**Pull request by benz-ubique**
Improve the qr code scanner so all devices can scann a qr code

### CODE REVIEW

File: common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QRCodeAnalyzer.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package ch.admin.bag.covidcertificate.common.qr
```
Comment (goebelUB) [inline common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QRCodeAnalyzer.kt:1]: Missing licence header

File: common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt [hunk 2]
```diff
@@ -160,60 +235,32 @@ abstract class QrScanFragment : Fragment() {
...
-		}
+	private fun setFlashAndButtonStyle() {
+		camera?.cameraControl?.enableTorch(isTorchOn)
+		val drawableId = if (isTorchOn) torchOnDrawable else torchOffDrawable
+		flashButton.isSelected = isTorchOn
+		flashButton.setImageResource(drawableId)
 	}
 
 	private fun handleInvalidQRCodeExceptions(qrCodeData: String?, error: Error?) {
+		//TODO maybe
```
Comment (goebelUB) [inline common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt:262]: TO be DOne when?

File: verifier/src/main/res/layout/fragment_qr_scan.xml [hunk 3]
```diff
@@ -16,11 +16,11 @@
...
 	tools:context=".qr.VerifierQrScanFragment">
 
-	<com.journeyapps.barcodescanner.DecoratedBarcodeView
-		android:id="@+id/barcode_scanner"
+	<androidx.camera.view.PreviewView
+		android:id="@+id/qr_code_scanner"
 		android:layout_width="match_parent"
 		android:layout_height="match_parent" />
-
+
```
Comment (goebelUB) [inline verifier/src/main/res/layout/fragment_qr_scan.xml:23]: Unnecessary whitespace -> reformat

File: common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt (hunk 2 above)
Comment (M-Wong) [reply common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt]: Before the next release, but we need this pull request for the testing session

**Review by M-Wong — APPROVED** [review]
